app/lib/actions/katello/repository_set/enable_repository.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Katello
     module RepositorySet
       class EnableRepository < Actions::EntryAction
+        middleware.use Actions::Middleware::BackendServicesCheck
```

Comment:
We could then also distinguish between different backends: the pulp actions needing just pulp to run, while candlepin actions the candlepin etc.